src/sentry/api/bases/organization.py

```diff
@@ -98,6 +98,17 @@ class OrganizationAndStaffPermission(StaffPermissionMixin, OrganizationPermissio
 class OrganizationAuditPermission(OrganizationPermission):
     scope_map = {"GET": ["org:write"]}
 
+    def has_object_permission(
+        self,
+        request: Request,
+        view: object,
+        organization: Organization | RpcOrganization | RpcUserOrganizationContext,
+    ) -> bool:
+        if super().has_object_permission(request, view, organization):
+            return True
+
+        return is_active_superuser(request)
```

Comment:
Can you add a comment here saying why we need this? To anyone who doesn't know about read/write this looks redundant